tests/testthat/_snaps/vignette-extending/extending.md

```diff
@@ -8,4 +8,112 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-This material has moved to <https://vctrs.r-lib.org/articles/pillar.html>.
+Tibbles are S3 objects of class `c("tbl_df", "tbl", "data.frame")`.
```

Comment:
Why is this in a snapshot test?

Reply:
Because the vignette output relies on other packages, and with that, I get an early alert if something changes.